.github/workflows/elvis_lint.yaml

```diff
@@ -0,0 +1,20 @@
+name: Elvis Linter
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          echo "$GITHUB_BASE_REF"
+          echo "$GITHUB_HEAD_REF"
+          set +e
+          curl -LO https://github.com/emqx/elvis/releases/download/1.0.0-emqx-1/elvis
+          chmod +x ./elvis
+          /usr/bin/git fetch --all
+          /usr/bin/git checkout $GITHUB_BASE_REF
+          /usr/bin/git checkout $GITHUB_HEAD_REF
```

Comment:
if we do not have to checkout the base, I believe there is no need to checkout the HEAD .